telegram/constants.py

```diff
@@ -21,6 +21,7 @@
 `Telegram Bots API <https://core.telegram.org/bots/api>`_.
 
 Attributes:
+    BOT_API_VERSION (:obj:`float`): `5.1`. Bot API version supported by PTB
```

Comment:
```suggestion
    BOT_API_VERSION (:obj:`float`): `5.1`. Current Telegram Bot API version supported by PTB
```

Reply:
```suggestion
    BOT_API_VERSION (:obj:`float`): `5.1`. Bot API version supported by this version of `python-telegram-bot`
```

Reply:
I liked the specified beginning of my suggestion and you just dropped it :(

Reply:
Because I didn't like too much … "Current Telegram Bot API version" sounds a bit too much like "this is the most recent version of the bot api & is supported by PTB" for my taste. I tried to instead emphasise that this is the version number that PTB supports, now matter if it's the latest or not.